modules/internal/ChapelDomain.chpl

```diff
@@ -2270,8 +2270,8 @@ module ChapelDomain {
       :type size: int
     */
     @unstable("createIndexBuffer() is subject to change in the future.")
-    inline proc createIndexBuffer(size: int) {
-      return _value.dsiCreateIndexBuffer(size);
+    inline proc createIndexBuffer(size: int, dataSorted:bool = false, isUnique:bool = false) {
```

Comment:
No need for explicit types if your defaults are clearly `bool`s. Whether you keep them or not, please make sure to fit this line into 80 columns.